index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Website</title>
+  </head>
+  <body>
+    <header>
+      <a class="name">Muh Noman</a>
+      <img class="navi" src="/images/navi.png" alt="navigation" />
+    </header>
+
+    <main>
+      <section>
+        <h1>Hey There. I’m Noman</h1>
+        <h6>I’am a Software Developer</h6>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
```

Comment:
- [ ] please make sure that the `font-family` used in this `p` tag matches the Figma. kindly look at the Figma design and use the same font-family

Your Design | Reference Design
-|-
![image](https://user-images.githubusercontent.com/18730106/164824606-00d1914c-d529-4ea0-b550-060efc30efbb.png)|![image](https://user-images.githubusercontent.com/18730106/164824712-3f70febf-ec7b-41d6-af45-b775afa88bdb.png)

Your Design | Reference Design
-|-
![image](https://user-images.githubusercontent.com/18730106/164827113-87a74e9b-d390-4f54-9ed2-0d2a76c9fff1.png)|![image](https://user-images.githubusercontent.com/18730106/164827266-8aa64727-7ffb-44d8-8269-68421e402999.png)

Reply:
@MrBrN197 Brother I try my best to get required the result of the font as in Figma but I think my CSS font family does not support this style. I want to show you through screenshots that I applied the same style but the results have not come as it in Figma.

![image](https://user-images.githubusercontent.com/48788360/164844952-9f1b1b2a-992e-44a3-8d5c-1ec449d4f678.png)
vs
![image](https://user-images.githubusercontent.com/48788360/164845015-3d7b43c0-db33-46be-800c-2d9bd549b999.png)